python/hail/expr/functions.py

```diff
@@ -853,6 +853,38 @@ def pl_dosage(pl) -> Float64Expression:
     return _func("plDosage", tfloat64, pl)
 
 
+@typecheck(pl=expr_array(expr_int32))
+def pl_to_gp(pl) -> ArrayNumericExpression:
+    """
+    Return the linear-scaled genotype probabilities from an array of Phred-scaled genotype likelihoods.
+
+    Examples
+    --------
+    .. doctest::
+
+        >>> hl.eval_expr(hl.pl_to_gp([0, 10, 100]))
+        [0.9090909090082644, 0.09090909090082644, 9.090909090082645e-11]
+
+
+    Notes
+    -----
+    This function assumes a uniform prior on the possible genotypes.
+
+    Parameters
+    ----------
+    pl : :class:`.ArrayNumericExpression` of type :py:data:`.tint32`
+        Array of Phred-scaled genotype likelihoods.
+
+    Returns
+    -------
+   :class:`.ArrayNumericExpression` of type :py:data:`.tfloat64`
+    """
+    max_phred_in_table = 2048
```

Comment:
How was 2048 chosen?

Reply:
totally arbitrary. This could actually be parameterized.

Reply:
there are routinely PLs that are 1800